content/en/events/2023/kcsna/schedule.md

```diff
@@ -9,6 +9,7 @@ weight: 3
 ## CFP
 
 A [CFP] for sessions is open through Friday, September 15th.
+Please, expect the schedule to be announced on October 6th, 2023.
```

Comment:
Let's mention all of the dates from the email to k-dev?

https://groups.google.com/a/kubernetes.io/g/dev/c/Gg5p-iyRaF0/m/DQkndQSMAwAJ

```
    CFP Opens: Tuesday August 15th

    CFP Closes: Friday, September 15th

    CFP Notification: Friday, September 29th

    Schedule Announced: Friday, October 6th
```